### ISSUE TITLE
[Problem] Edit Custom Property and sorting

### ISSUE THREAD

**Issue by kaiwas** [FreeCAD/FreeCAD#12837]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Sometimes I have to write my own properties. If there are a lot of them, unfortunately there are mistakes in their writing. But at the moment there is no way to quickly correct the inaccuracy. Because "edit property" is missing.

You have to delete the previously created property and create it again to correct the error.

If there was an “edit property” tab that would open a window similar to the one that opens when you click “add property,” but with the data for this property already filled in previously, that would be very convenient. (picture number 1).
It would be possible to change the property name and its description (the group can be changed now)

![Scr_20240310_113135](https://github.com/FreeCAD/FreeCAD/assets/32273833/b74df4d3-bff2-468e-9da1-cdafe46f0a61)

Another problem is the order of properties in the panel. Now there is no way to sort them in the desired order (the order is strictly alphabetical). We have to come up with various workarounds so that the necessary properties are in the right order and are always at the top of the panel (picture number two)

![Scr_20240310_113655](https://github.com/FreeCAD/FreeCAD/assets/32273833/e5dbd3e1-8946-439e-9edb-1d7b8bc81444)

### Full version info

```shell
Any version
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
related: #12120

**Comment by kaiwas** [comment]
> related: #12120

Sorry if I created a duplicate. I searched for the problem, but it seems I used the wrong terms in the search.
Thank you

**Comment by maxwxyz** [comment]
I think it is not a duplicate but still related. Currently you cannot sort or edit the name of the properties as they are currently implemented. @pieterhijma ping to consider this as well.

**Comment by pieterhijma** [comment]
Yes, very good point. It's not a duplicate. I'll look into that.

**Comment by pieterhijma** [comment]
Changing the order appears to be a much requested feature, I know of two others who would like this.

**Comment by kaiwas** [comment]
> Changing the order appears to be a much requested feature, I know of two others who would like this.

I had another idea that would significantly improve access to properties.
But this topic was closed and moved to discussions that no one will ever see )
https://github.com/FreeCAD/FreeCAD/discussions/14026

**Comment by furgo16** [comment]
Also related, but for VarSets: https://github.com/FreeCAD/FreeCAD/issues/16222
